3.10/graphs-enterprise-graphs-getting-started.md

```diff
@@ -127,8 +128,23 @@ EnterpriseGraphs.
 Second, because you have changed the name of the `_from` collection, you need
 to provide a `from-collection-prefix`. The same is true for the `_to` collection.
 
-Note that you can only change all vertices on `_from` respectively `_to`
-side with this mechanism. However, you can use different collections on `_from` and `_to`.
+Note that this mechanism does not provide the option to selectively replace
+collection names. It only allows replacing all collection names on `_from` 
+respectively `_to`.
+This means that, even if you use different collections on `_from` and `_to`, 
+their names are modified based on the prefix that is specified.  
+
+Consider the following example where `_to:users_vertices/Bob` points to a
+different collection. When using `--to-collection-prefix "vertices"` to rename
+the collections, all collections names on the `_to` side are renamed to
+`vertices` as this transformation solely allows for the replacement of all
+collection names within a column.
+
+```
+{"_key":"121","_id":"old_edges/121","_from":"old_vertices/Bob","_to":"old_vertices/Charly","_rev":"_edwW20----","attribute2":"value2"}
+{"_key":"122","_id":"old_edges/122","_from":"old_vertices/Charly","_to":"old_vertices/Alice","_rev":"_edwW20G---","attribute2":"value3"}
+{"_key":"120","_id":"old_edges/120","_from":"old_vertices/Alice","_to":"users_vertices/Bob","_rev":"_edwW20C---","attribute2":"value1"}
```

Comment:
I'm confused about what this is supposed to show. Two of the `_to` attributes have `old_vertices` as the collection, one has `old_vertices`. My understanding is that if the value already has `<some-string>/` as the prefix, then `--to-collection-prefix` does not prepend a collection name, unless you use `--overwrite-collection-prefix true`, but this is explained after the example documents...

While not JSON but JSONL, I believe the code highlighting works anyway. Some whitespace for readability:
```suggestion
```json
{"_key":"121", "_id":"old_edges/121", "_from":"old_vertices/Bob", "_to":"old_vertices/Charly", "_rev":"_edwW20----", "attribute2":"value2"}
{"_key":"122", "_id":"old_edges/122", "_from":"old_vertices/Charly", "_to":"old_vertices/Alice", "_rev":"_edwW20G---", "attribute2":"value3"}
{"_key":"120", "_id":"old_edges/120", "_from":"old_vertices/Alice", "_to":"users_vertices/Bob", "_rev":"_edwW20C---", "attribute2":"value1"}
```